app/page.tsx

```diff
@@ -99,113 +99,115 @@ export default function Home() {
+                </div>) : (
+                <p className={infoMessageClass}>
+                  Good Job! There are no more pending live classes today.
+                </p>
+              )}
+            </div>
+            <div className="bg-heading">
+              <h1 className="text-primary ml-4 font-semibold text-xl">Tests</h1>
+              {quizzes.length > 0 ? (
+                <div className="grid grid-cols-1 gap-4 pb-40">
+                  {quizzes.map((data, index) => (
+                    <div key={index} className="flex mt-4 items-center" >
+                      <div>
+                        <p className={`${commonTextClass}`}>
+                          {formatSessionTime(data.sessionOccurrence.start_time)}
+                        </p>
+                        <p className={`${commonTextClass}`}>
+                          {formatSessionTime(data.sessionOccurrence.end_time)}
+                        </p>
+                      </div>
+                      <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
+                        <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0  rounded-s-md`}></div>
+                        <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
+                          <span className="font-normal pr-4">Subject:</span> {data.sessionDetail.meta_data.stream}
+                          <div className="text-sm md:text-base font-semibold ">
+                            <span className="font-normal pr-9">Type:</span> {data.sessionDetail.meta_data.test_type}
+                          </div>
                         </div>
+                        {renderButton(data)}
                       </div>
-                      {renderButton(data)}
                     </div>
-                  </div>
-                ))}
-              </div>) : (
-              <p className={`${infoMessageClass}`}>
-                Good Job! There are no more pending tests today.
-              </p>
-            )}
-          </div>
-          <div className="text-blue-500 underline pb-40 bg-heading text-center px-4">
-            <Link href="https://docs.google.com/spreadsheets/d/1zIjYf4KUXkAHLiuWdtHp-l-5xa96obBtbQU0UvuOy7w/edit?usp=sharing" target="_blank">
-              Click on the link to see all upcoming Tests
-            </Link>
-          </div>
-          <BottomNavigationBar />
-        </main>)}
-      {/* ) : (
+                  ))}
+                </div>) : (
+                <p className={`${infoMessageClass}`}>
+                  Good Job! There are no more pending tests today.
+                </p>
+              )}
+            </div>
+            <div className="text-blue-500 underline pb-40 bg-heading text-center px-4">
+              <Link href="https://docs.google.com/spreadsheets/d/1zIjYf4KUXkAHLiuWdtHp-l-5xa96obBtbQU0UvuOy7w/edit?usp=sharing" target="_blank">
```

Comment:
What is this Google Sheet? Now that we have userID, we still need it?

Reply:
`Test Planner` link is required. We were even thinking to add `Course planner` link here.

Reply:
But they will be different for different batches, so can't be hardocded here

Reply:
I'll remove this code